### PR TITLE
f strings in core/sorting

### PR DESCRIPTION
**Pull request by donehower**
- [ ] references #29547
- [ ] tests added / passed
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/core/sorting.py [hunk 1]
```diff
@@ -208,7 +208,7 @@ def lexsort_indexer(keys, orders=None, na_position="last"):
             cat = Categorical(key, ordered=True)
 
         if na_position not in ["last", "first"]:
-            raise ValueError("invalid na_position: {!r}".format(na_position))
+            raise ValueError(f"invalid na_position: {na_position}")
```
Comment (WillAyd) [inline pandas/core/sorting.py]:
```suggestion
            raise ValueError(f"invalid na_position: {repr(na_position)}")
```

File: pandas/core/sorting.py [hunk 2]
```diff
@@ -264,7 +264,7 @@ def nargsort(items, kind="quicksort", ascending: bool = True, na_position="last"
     elif na_position == "first":
         indexer = np.concatenate([nan_idx, indexer])
     else:
-        raise ValueError("invalid na_position: {!r}".format(na_position))
+        raise ValueError(f"invalid na_position: {na_position}")
```
Comment (WillAyd) [inline pandas/core/sorting.py]:
```suggestion
        raise ValueError(f"invalid na_position: {repr(na_position)}")
```

**Review by WillAyd — CHANGES_REQUESTED** [review]
similar comment to other PR about merging master. Thanks again!

File: pandas/core/sorting.py (hunk 1 above)
Comment (jreback) [reply pandas/core/sorting.py]: @WillAyd these seem excessive, adding repr on a boolean is not worth IMHO

**Comment by jreback** [comment]
@donehower pls merge master

File: pandas/core/sorting.py (hunk 1 above)
Comment (WillAyd) [reply pandas/core/sorting.py]: Ah makes sense. Oversight on my end

**Review by WillAyd — APPROVED** [review]

**Comment by WillAyd** [comment]
Thanks @donehower !

**Pull request merged**